src/pages/settings.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, ScrollView } from 'react-native';
+import { RoutePropsHelper } from '../router';
+import { Center } from '../components/center';
+import { Divider, Switch, Paragraph, Chip, Text } from 'react-native-paper';
+import { ThemeProvider } from '@react-navigation/native';
+import { paperTheme } from '../theme/paper-theme';
+import { Button, TextInput } from 'react-native-paper';
+import { initialWindowSafeAreaInsets } from 'react-native-safe-area-context';
+import { ContentPadding } from '../components/content-padding';
```

Comment:
```suggestion
import React, { useState } from 'react';
import { StyleSheet, View, ScrollView } from 'react-native';
import { RoutePropsHelper } from '../router';
import { TextInput } from 'react-native-paper';
import { ContentPadding } from '../components/content-padding';
import { Button } from '../components/button';
```
Remove unused components and import Adam's button.